.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,34 @@
+name: "Close Stale Issues"
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */4 * * *"
+
+jobs:
+  cleanup:
+    permissions:
+      issues: write
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+      - uses: actions/stale@v8
+        with:
+          issue-types: issues
+          stale-issue-message: This issue has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+
+          # These labels are required
+          stale-issue-label: Stale
+          exempt-issue-labels: No Autoclose
+
+          # Issue timing
+          days-before-stale: 30
+          days-before-close: 7
+
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          loglevel: DEBUG
+          # Set dry-run to true to not perform label or close actions.
+          dry-run: false
```

Comment:
Could we merge with `dry-run` set to true first to see what happens, and then flip to true after review?